client/src/components/Projects/URLs/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faTwitter,
+  faDiscord,
+  faTelegram,
+  faMedium,
+} from "@fortawesome/free-brands-svg-icons";
+import { ProjectURLs } from "src/entities/project.entities";
+import { IconProp } from "@fortawesome/fontawesome-svg-core";
+
+const URLs = ({ projectURLs }: { projectURLs: ProjectURLs }) => {
+  const render = (icon: IconProp, style: string, url?: string) => {
+    return url ? (
+      <a href={url} className={`${style} m-1`}>
+        <FontAwesomeIcon icon={icon} />
+      </a>
+    ) : null;
+  };
+
+  const medium = render(faMedium, "text-medium", projectURLs.medium);
+  const twitter = render(faTwitter, "text-twitter", projectURLs.twitter);
+  const discord = render(faDiscord, "text-discord", projectURLs.discord);
+  const telegram = render(faTelegram, "text-telegram", projectURLs.telegram);
+  const cardanoScan = null;
+  const poolpm = null;
+
+  return (
+    <div>
```

Comment:
it is better to use flexbox for space between elements. You can do

<div className="flex flex-col gap-4">
  { map function }
</div>
using this you don't need to manually add margin to each box, hence less place to change, when needed.

the standard for flex gap is :

gap-2: for text, e.g., space between icon and text
gap-4: space between bigger component (between buttons etc.)
gap-8: space between title and page